raptorWeb/raptorbot/discordbot/util/raptorbot_settings.py

```diff
@@ -0,0 +1,12 @@
+from os import getenv
+from dotenv import load_dotenv
+
+# Load .env file and load Bot Token from file
+load_dotenv()
```

Comment:
the TOKEN here should be moved to the .env file for the whole project. We may want to move that current .env file from /config/db to the raptorWeb project root

Reply:
This will be addressed later on, when a very large project directory re-structure will occur.